From e50e4695c9d906227a6d9a04025a797e58aa2a6f Mon Sep 17 00:00:00 2001
From: Matthew Fatheree <Matthew.Fatheree@belkin.com>
Date: Fri, 16 May 2014 22:06:21 +0700
Subject: [PATCH 31/34] Enable anonymous mount by default for automount

when using /etc/uci-defaults/fstab to generate /etc/config/fstab we
observed issue with auto-share. The /etc/config/fstab is created after
the procd executes hotplug function, which makes the cold plug devices
failed to be added to samba (this only happened in case of factory
reset). We created a static /etc/config/fstab function to avoid this
scenario

Signed-off-by: Matthew Fatheree <Matthew.Fatheree@belkin.com>
---
 package/linksys-base-files/files/etc/config/fstab |    7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 package/linksys-base-files/files/etc/config/fstab

diff --git a/package/linksys-base-files/files/etc/config/fstab b/package/linksys-base-files/files/etc/config/fstab
new file mode 100644
index 0000000..31a001b
--- /dev/null
+++ b/package/linksys-base-files/files/etc/config/fstab
@@ -0,0 +1,7 @@
+config global
+        option anon_mount '1'
+        option anon_swap '0'
+        option auto_swap '1'
+        option auto_mount '1'
+        option delay_root '5'
+        option check_fs '0'
-- 
1.7.9.5

